tests/Doctrine/Tests/Common/Persistence/Mapping/SymfonyFileLocatorTest.php

```diff
@@ -62,6 +62,90 @@ public function testGetAllClassNames()
         $this->assertEquals(array("Foo\\stdClass"), $locator->getAllClassNames("global"));
     }
 
+
+    public function customNamespaceSeparatorProvider()
+    {
+        return array(
+            'directory separator' => array(DIRECTORY_SEPARATOR, "/_custom_ns/dir"),
+            'underscore separator' => array('_', "/_custom_ns/underscore"),
```

Comment:
Yes, I know this. So now reference to underscore is removed from everywhere. It's only possibly to fault when end user manually sets underscore as namespace separator when storing mappings.